packages/upswyng-web/src/components/__tests__/Resource.test.tsx

```diff
@@ -25,27 +32,18 @@ jest.mock("../../App.styles", () => ({
     },
   },
 }));
-jest.mock("react-router-dom", () => ({
-  useParams: () => ({ resourceId: "some resource ID" }),
-}));
-jest.mock("../PageBanner", () => "PageBanner");
-jest.mock("../Schedule", () => "Schedule");
-jest.mock("../Map", () => "Map");
-jest.mock("../useResource.tsx");
-const mockedUseResource = useResource as jest.Mock;
 
-describe("<Resource/>", () => {
-  mockedUseResource.mockImplementation(() => foodResource) as typeof jest.mock;
-  const wrapper = shallow(<Resource />);
+const mockUseResource = useResource as jest.Mock;
 
-  it("renders the charityname property of the resource prop object", () => {
-    expect(wrapper.find("PageBanner").prop("text")).toBe(foodResource.name);
-  });
```

Comment:
I removed this test (I originally wrote 😅) because it didn't really provide much value. A better option would be to either:

- actually render out the `<PageBanner/>`, which is currently prevented by the mock of it towards the top of this file
- or opt for what I did here, and don't worry about testing `<PageBanner/>` within the test of `<Resource/>`